implementations/micrometer-registry-graphite/src/main/java/io/micrometer/graphite/GraphiteMeterRegistry.java

```diff
@@ -78,20 +77,26 @@ private static GraphiteReporter defaultGraphiteReporter(GraphiteConfig config, C
     }
 
     public void stop() {
-        this.reporter.stop();
+        if (this.config.enabled()) {
+            this.reporter.stop();
+        }
     }
 
     public void start() {
-        this.reporter.start(config.step().getSeconds(), TimeUnit.SECONDS);
+        if (this.config.enabled()) {
+            this.reporter.start(config.step().getSeconds(), TimeUnit.SECONDS);
+        }
     }
 
     @Override
     public void close() {
-        if (config.enabled()) {
-            reporter.report();
+        if (this.config.enabled()) {
```

Comment:
Different from the Spring Boot style, I don't love the excess `this` statements, and prefer `this` to be used only when necessary.

Reply:
I've let it go to save time for a while, but thought it worth mentioning.

Reply:
@jkschneider Ah, sorry. I didn't know the convention here. I prefer to use `this` for fields explicitly as it makes me easily notice they're class member fields without IDE like in GitHub. If you still disagree, I'll revert the change and be happy to comply. Just let me know.

Reply:
I do understand the reasoning, but still have a preference for dropping extraneous `this` prefixes. I know it's subjective, but personally I find the cost of extra verbosity in the place where I spend most of my time (in the IDE) not worth the benefit of field differentiation where I spend relatively little time (browsing in Github).

Reply:
@jkschneider Thanks for the feedback! I'll try to stick with the convention unless I'm confused with other projects. If I break it, just poke me to realise. Then I'll polish to comply. I did the opposite polish for the `this` keyword and hope you're happy with the change now 😄

Reply:
Thanks for the understanding!